### PR TITLE
fix cloudcore crash when nodekeepalivechannel is nil

### PR DESCRIPTION
**Pull request by lvfei103650**
**What type of PR is this?**

/kind bug

**What this PR does / why we need it**:
fix cloudcore crash when nodekeepalivechannel is nil

**Which issue(s) this PR fixes**:

Fixes #

**Special notes for your reviewer**:

**Does this PR introduce a user-facing change?**:
NONE

### CODE REVIEW

**Comment by lvfei103650** [comment]
image

**When the network condition is very weak and edgenode keeps re-registering, cloudcore will crash in here**

**Comment by fisherxu** [comment]
Thanks for the fixing, will have a review :) And kindly reminder, looks like github can't identify your email addr in the commit, you need to associate this Email addr to your github personal setting.
image

File: cloud/pkg/cloudhub/handler/messagehandler.go [hunk 1]
```diff
@@ -166,7 +166,11 @@ func (mh *MessageHandle) OnRegister(connection conn.Connection) {
 // KeepaliveCheckLoop checks whether the edge node is still alive
 func (mh *MessageHandle) KeepaliveCheckLoop(info *model.HubInfo, stopServe chan ExitCode) {
 	keepaliveTicker := time.NewTimer(time.Duration(mh.KeepaliveInterval) * time.Second)
-	nodeKeepaliveChannel, _ := mh.KeepaliveChannel.Load(info.NodeID)
+	nodeKeepaliveChannel, ok := mh.KeepaliveChannel.Load(info.NodeID)
+	if !ok {
+		klog.Errorf("fail to load node %s", info.NodeID)
+		return
```
Comment (fisherxu) [inline cloud/pkg/cloudhub/handler/messagehandler.go:172]: Looks good, but it should be not correct that just return here, and we can fix it in follow-up PRs.

**Pull request merged**